### PR TITLE
Fixed the missing state variable bug.

### PR DESCRIPTION
**Pull request by JoschaO**
Not as clean as I would like, but it works, by making a new list and
adding a checked variable to it. I thought ngModel would take care of
that, but it didn't. This still works and is fairly clean.

Needs a manual merge, as there is a little overlap in app.module.ts

### CODE REVIEW

**Comment by JoschaO** [comment]
Fixed Merging issues with ticket db being broken, and checking tickets before displaying them.

**Comment by JoschaO** [comment]
This will make dev work again properly, so pushing this out should happen soon

File: TicketDesk/ngWebClient-StandAlone/src/app/ticket-list/ticket-list.component.ts [hunk 1]
```diff
@@ -11,21 +11,27 @@ import { FormsModule } from '@angular/forms';
...
+  private displayList: {'ticket': Ticket, 'checked': boolean}[];
   @Input() ticketListResults: { 'ticketList': Ticket[], 'maxPages': number };
   @Input() columns: string[];
   currentPage: number;
   ngOnInit() {
-    // State is created from ngModal in Angular
-    this.ticketListResults.ticketList.forEach(x => x.state = false);
+    this.displayList = [];
+    for (const ticket of this.ticketListResults.ticketList) {
+      if (ticket) { // ticket is of type ticket
```
Comment (kixiQu) [inline TicketDesk/ngWebClient-StandAlone/src/app/ticket-list/ticket-list.component.ts:22]: that isn't what this checks, is it? seems like some kind of filter then mapping an arrow function would work better here. not worth changing for the review, though.

**Review by kixiQu — APPROVED** [review]
As discussed

**Pull request merged**